x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate/analytics_collection_integrate_javascript_embed.tsx

```diff
@@ -90,25 +88,7 @@ window.elasticAnalytics.createTracker({
           <p>
             <FormattedMessage
               id="xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.javascriptEmbed.stepThree.description"
-              defaultMessage="Track individual search events, like result clicks and searches, by using the trackSearch or trackSearchClick methods. {link}"
```

Comment:
Question: Is removing the link `Learn more about tracking events` from here an intentional change?

Reply:
That one from this thread https://elastic.slack.com/archives/C042WRLGS31/p1684259120422179